extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/HibernateOrmRuntimeConfig.java

```diff
@@ -23,6 +26,14 @@ public interface HibernateOrmRuntimeConfig {
     @ConfigDocMapKey("persistence-unit-name")
     Map<String, HibernateOrmRuntimeConfigPersistenceUnit> persistenceUnits();
 
+    /**
+     * Configuration for enabling request scoped sessions
+     */
+    @ConfigDocSection
+    @WithName("request-scoped.enable")
```

Comment:
I think this shouldn't be `request-scoped.enable`, but `request-scope.enabled`, because `enabled` is what we use everywhere else.

```suggestion
    @WithName("request-scope.enabled")
```

Reply:
Actually it seems you used `request-scoped.enabled`, while I was suggesting `request-scope.enabled`... I'll fix that.

Reply:
... on second thought Reactive Messaging already uses `request-scoped.enabled`, so you're right, let's go with that.